jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+// Configure the build properties
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '500', daysToKeepStr: '4')),
+    disableConcurrentBuilds(),
+])
+
+pipeline {
+
+  // env variables
+  environment { 
+
+    jslint_check = "gitarro.ruby2.1  -r SUSE/spacewalk" + 
+                " -c javascript_lint -d \"javascript-lint\" " +
+                " -f \".js\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
+                " -g ${env.WORKSPACE} " +
+                "--check " // --changed_since 3600 
```

Comment:
if we comment out this we will reach the Rate limit of github ( `changed_since`)